ui/app/styles/components/replication-dashboard.scss

```diff
@@ -0,0 +1,71 @@
+.replication-dashboard {
+  .has-margin-bottom {
+    padding-bottom: 9px;
+  }
+
+  .has-large-margin-top {
+    margin-top: 40px;
+  }
+
+  .selectable-card {
+    line-height: normal;
+
+    .title-number {
+      font-size: 28px;
+      align-self: end;
+    }
+  }
+
+  .helper-text {
+    font-weight: normal;
+  }
+
+  .selectable-card-container {
+    display: grid;
+    grid-template-columns: repeat(auto-fit, minmax(400px, 1fr));
+    grid-template-rows: 1fr;
+    grid-gap: 2rem;
+
+    .card-container {
+      display: grid;
+      grid-template-columns: 1fr 1fr;
+      grid-template-rows: 1fr 1fr 0.8fr;
+      padding: $spacing-l 0 18px $spacing-l;
+
+      .grid-item-title {
```

Comment:
i am a total newbie to css grid, so at the risk of sounding ignorant could you tell me more about the benefits of using css grid inside the cards here? it seems like an awful lot of css for a view that isn't so complex. what's your thinking on grid vs. having the cards with 2 divs sitting side by side? eager to hear more since you have much more expertise in this than i do!

Reply:
Yeah, let's chat.  What I love about css grid, is it allows me to not only have the columns but the rows a certain height, helping keep a separate area for each of the different chunks of information we are presenting.  Instead of fussing with margins, I just have defined row heights that are fractions of each other.